cfme/automate/service_dialogs.py

```diff
@@ -64,7 +64,8 @@
         version.LOWEST: Input("field_default_value"),
         "5.5": AngularSelect("field_default_value")}),
     ('dynamic_chkbox', Input("field_dynamic")),
-    ('apply_btn', '//a[@title="Apply"]')
+    ('apply_btn', '//a[@title="Apply"]'),
+    ('ele_name', Input("field_name"))
```

Comment:
Automation is behaving weirdly since last two builds . Selecting a value in drop down ( ('choose_type' - line 48) wipes out values filled earlier in text box. So I have moved them below so that the values are filled at the last .